.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
 
 # Commitizen enforces semantic and conventional commit messages.
 - repo: https://github.com/commitizen-tools/commitizen
-  rev: 3.5.2
+  rev: v3.13.0
   hooks:
```

Comment:
Thanks for making the changes @behnazh-w ! 
I wonder if you have an idea why this error starts to appear given the tag `3.5.2` we have been using works fine so far. I had a look at the commitizen repository at https://github.com/commitizen-tools/commitizen/tags and I saw that all of their tags are now in the format of `v<release_version>`.
Did the maintainers of commitizen decided to update all the tag names for their releases? :thinking:

Reply:
Most likely, but I can't tell for sure how they have renamed the tag.